dev/test-studio/migrations/cleanup-empty/cleanup-empty-values.ts

```diff
@@ -4,7 +4,7 @@ import {unset} from 'sanity/migrate/mutations'
 
 export default defineMigration({
   name: 'Cleanup empty values',
-  documentType: 'species',
+  documentType: ['playlist', 'species'],
```

Comment:
Should we make it read `documentTypes` and always require it to be an array? Making it an array helps with discoverability, e.g. if you see `documentTypes: ['playlist']` it's immediately clear that you can migrate more than one type of document at the same time.